components/places/src/import/fennec/bookmarks.rs

```diff
@@ -303,11 +303,11 @@ lazy_static::lazy_static! {
             isLocal
         )
         SELECT
-            normalize_root_guid(b.guid),
+            normalize_root_guid(sanitize_utf8(b.guid)),
             b.type,
-            (SELECT normalize_root_guid(p.guid) FROM fennec.bookmarks p WHERE p._id = b.parent),
+            IFNULL((SELECT normalize_root_guid(sanitize_utf8(p.guid)) FROM fennec.bookmarks p WHERE p._id = b.parent), 'unfiled_____'),
             b.position,
-            b.title,
+            sanitize_utf8(b.title),
```

Comment:
We don't want to do `keyword` or `tags` below?

Reply:
We don't need tags because we drop them. I thought that was true for `keyword` too, but it's not - that is put into the moz_bookmarks_synced table. So tl;dr, yes, I should! Thanks!